lib/transaction/transaction.js

```diff
@@ -231,24 +217,37 @@ Transaction.prototype._isFeeDifferent = function() {
     var fee = this._fee;
     var unspent = this._getUnspentValue();
     if (fee !== unspent) {
-      return 'Unspent value is ' + unspent + ' but specified fee is ' + fee;
+      return new errors.Transaction.FeeError.Different(
+          'Unspent value is ' + unspent + ' but specified fee is ' + fee);
```

Comment:
Wow, that looks super weird to me :). But this is not my project so I will adapt. But do you think you can make the linter say these things? That would make it easier to adopt the correct coding style.

Actually, this fits in 120 characters so I just put it all on one line.